client/landing/stepper/declarative-flow/start-writing.ts

```diff
@@ -28,13 +29,17 @@ const startWriting: Flow = {
 
 	useStepNavigation( currentStep, navigate ) {
 		const flowName = this.name;
-		function submit( providedDependencies: ProvidedDependencies = {} ) {
+		async function submit( providedDependencies: ProvidedDependencies = {} ) {
 			recordSubmitStep( providedDependencies, '', flowName, currentStep );
 			switch ( currentStep ) {
 				case 'site-creation-step':
 					return navigate( 'processing' );
 				case 'processing': {
 					if ( providedDependencies?.siteSlug ) {
+						await updateLaunchpadSettings( String( providedDependencies?.siteSlug ), {
```

Comment:
Making a note for myself: `updateLaunchpadSettings` itself has a `.catch()`, so if this API fails, it will just continue to move on to the next line of code.